config/routes.rb

```diff
@@ -1,9 +1,9 @@
 Rails.application.routes.draw do
   root 'static_pages#home'
-  get 'static_pages/home'
-  get 'static_pages/help'
-  get 'static_pages/about'
-  get 'static_pages/contact'  
+  get '/help', to: 'static_pages#help'
```

Comment:
There are so many different ways to define routes in rails ... but I think this pattern is fine 🤷🏻‍♂️  just stay consistent I guess